formats/metadata.json

```diff
@@ -60,6 +60,19 @@
       "items": {
         "$ref": "#/definitions/route"
       }
+    },
+    "access_limited": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": [ "users" ],
+      "properties": {
+        "users": {
```

Comment:
I would have preferred to have it named "user_uids" or similar so that is explicit what it contains and to make it cleaner if we needed to migrate to passing around more data about each user.